app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -593,6 +593,17 @@ class PatientRepository @Inject constructor(
         .toObservable()
   }
 
+  fun deleteBusinessId(businessId: BusinessId): Completable {
```

Comment:
Should we add an instrumented test for this?

Reply:
I didn't add a test since it was a straight-forward operation. I will add it now.

Reply:
Thanks, to me it's just mainly about adding a regression test to ensure that the record timestamps get updated as expected more than anything else.